types/use-prefers-color-scheme/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for use-color-scheme 0.1
+// Project: https://github.com/rfoel/use-prefers-color-scheme
+// Definitions by: Marton Lederer <https://github.com/martonlederer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function usePrefersColorScheme(): "no-preference" | "light" | "dark";
+export = usePrefersColorScheme;
```

Comment:
Based on the underlying library, it seems like you want `export default usePrefersColorScheme`.